www/css-utils/card/readme.md

```diff
@@ -74,31 +64,29 @@ To apply card css utils, we can use 'fw-card-#{$elevation}'
   }
 </style>
 <template>
-  <div>
-    <div class="fw-card-1 fw-py-16 fw-px-20 fw-flex fw-flex-row">
-      <div>
-        <fw-avatar
-    size="medium"
-    image="https://images.unsplash.com/photo-1529778873920-4da4926a72c2?ixlib=rb-1.2.1&auto=format&fit=crop&w=300&q=80"
-  ></fw-avatar>
-      </div>
-      <div class="fw-flex-grow fw-px-16 fw-flex fw-flex-column">
-        <div class="fw-type-h5 fw-mb-8">Courtney Henry</div>
-        <div class="fw-type-xs fw-mb-16">Hey! I need help with cancellation of a combo pack that I had ordered yesterday.</div>
-        <div class="fw-type-xs fw-flex fw-flex-row options">
-          <div class="fw-pr-12">
-            <span class="fw-mr-8"><fw-icon name="inbox" size="12" library="system" ></fw-icon></span>
-            <span>Cancellation</span>
-          </div>
-          <div class="fw-px-12">Open</div>
-          <div class="fw-px-12">Due in 1 hr</div>
-          <div class="fw-px-12">Alex James (Me)</div>
-        </div>
-      </div>
-      <div>
-        <span class="fw-type-xs">11 May 2021, 5:30 PM</span>
-      </div>
-    </div>
+  <div class="fw-card-1 fw-py-16 fw-px-20 fw-flex fw-flex-row">
```

Comment:
can you also change it to section

Reply:
Same as https://github.com/freshworks/crayons/pull/629/files#r901594563.